helm-chart/amalthea/values.yaml

```diff
@@ -76,10 +76,10 @@ extraChildResources: []
   #   group: amalthea.dev
 
 image:
-  repository: renku/amalthea
+  repository: degano/amalthea
   pullPolicy: IfNotPresent
   # Overrides the image tag whose default is the chart appVersion.
-  tag: "0.0.1-rc1"
+  tag: '0.1.0-rc11.n006.h46d1a71'
```

Comment:
Same as the previous comment about the chart tag. I always forget and check this in with chartpress.